nextflow.config

```
@@ -390,7 +390,7 @@ manifest {
     description     = """An open-source analysis pipeline to detect germline or somatic variants from whole genome or targeted sequencing"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '3.5dev'
+    version         = '3.14.1'
```

Minor fix needed here.
```suggestion
    version         = '3.4.1'
```